OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -19,7 +19,7 @@ public class TurretedInfo : ITraitInfo, UsesInit<TurretFacingInit>, Requires<Bod
 	{
 		public readonly string Turret = "primary";
 		[Desc("Rate of Turning")]
```

Comment:
There is another instance of this though. So yeah, all descs should be unified as part of this PR imo.